src/registrar/templates/domain_detail.html

```diff
@@ -46,8 +46,11 @@ <h2 class="margin-top-neg-1"> DNS name servers </h2>
     {% include "includes/summary_item.html" with title='Your contact information' value=request.user.contact contact='true' edit_link=url %}
 
     {% url 'domain-security-email' pk=domain.id as url %}
-    {% include "includes/summary_item.html" with title='Security email' value=domain.security_email edit_link=url %}
-
+    {% if security_email is not None and security_email != "dotgov@cisa.dhs.gov"%}
```

Comment:
just browsing really quick, everything else on a fast inspection looks great. Here can we have this default email just pull from PublicContact.getDefaultSecurityEmail().email or even do domain.getDefaultSecurityContact().email? That way if the default changes we dont have to remember to change it in multiple places